webapp/channels/src/components/select_team/components/select_team_item.tsx

```diff
@@ -58,18 +57,16 @@ export default class SelectTeamItem extends React.PureComponent<Props> {
         let icon;
         if (loading) {
             icon = (
-                <LocalizedIcon
+                <span
                     className='fa fa-refresh fa-spin right signup-team__icon'
-                    component='span'
-                    title={{id: t('generic_icons.loading'), defaultMessage: 'Loading Icon'}}
```

Comment:
Since this component type was span , we create a `span` tag instead of `i` tag we created in other component